pom.xml

```diff
@@ -27,7 +27,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci</groupId>
     <artifactId>jenkins</artifactId>
-    <version>1.37</version>
+    <version>1.38-SNAPSHOT</version>
```

Comment:
(CI builders will not let you an unmarked snapshot, and it is very risky generally.)